ros_ws/src/simulation/racer_description/urdf/racer.xacro

```diff
@@ -3,6 +3,10 @@
 <robot name="myrobot" 
   xmlns:xacro="http://www.ros.org/wiki/xacro">
 
+  <xacro:arg name="use_gpu" default="true"/>
+  <xacro:arg name="visualize_lidar" default="false"/>
+  <xacro:arg name="laser_sample_count" default="1080"/>
+
   <xacro:include filename="$(find racer_description)/urdf/racer_plugins.gazebo" />
```

Comment:
Now that this xacro file is used outside of gazebo, I think it makes sense to make this include optional.
```xml
<xacro:arg name="use_gazebo" default="false"/>
<xacro:if value="$(arg use_gazebo)">
    <xacro:include filename="$(find racer_description)/urdf/racer_plugins.gazebo" />
</xacro:if>
```

Reply:
Does it matter though? The content of racer_plugins is only used by gazebo anyway, making an argument to include that file is pretty much useless.